src/pages/api/orgs/[orgId]/events.ics/index.ts

```diff
@@ -0,0 +1,74 @@
+import dayjs, { Dayjs } from 'dayjs';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import BackendApiClient from 'core/api/client/BackendApiClient';
+import { ZetkinEvent } from 'utils/types/zetkin';
+
+const timeStamp = (t: Dayjs): string => t.format('YYYYMMDDTHHmmss');
+const formatString = (str: string): string =>
+  str
+    .replaceAll('\n', '\\n')
+    .match(/.{1,75}/g)
+    ?.join('\n\t') ?? '';
+
+export default async function handle(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
+  const { orgId } = req.query;
+
+  const apiClient = new BackendApiClient(req.headers);
+
+  const startTime = dayjs()
+    .subtract(2, 'month')
+    .toDate()
+    .toISOString()
+    .slice(0, 10);
+
+  const events = await apiClient.get<ZetkinEvent[]>(
+    `/api/orgs/${orgId}/actions?frecursive&ilter=start_time>=${startTime}`
+  );
+
+  const vEvents: string[] = [];
+  events
+    .filter((event) => !event.cancelled && event.published)
+    .forEach((event) => {
+      vEvents.push(`BEGIN:VEVENT`);
+      vEvents.push(`UID:${event.id}@zetkin.org`);
```

Comment:
Could we use the environment variable for the host here as well? Or maybe the API domain (which is where the ownership of the event truly lies)?

Reply:
Changed it to the env. variable. I think the idea is that it's globally unique, but not displayed to the user.

Reply:
Great! Confirmed fixed in 289fa0d3b5b7a700da3a410035ee1858ffa5914a.